README.md

```diff
@@ -30,6 +30,14 @@ native layer.
 - You can just run the following command to test.
   - ```./bacardi test```
 
+## Lint
```

Comment:
I think this is unnecessary. We already have `format` command in our README.md.

Reply:
IMHO, I think we need to introduce lint command. lint and format command are different

Reply:
@romandev, if you think so, why don't we discuss about ISSUE #177 ?

Reply:
@yjaeseok Yeah, they are usually each different commands. But the current implementation of `lint` command is implemented based on `format` command. So, there is no difference between two commands.

If there are some lint errors, users can run `./bacardi format` command.
So, all lint errors will be fixed. Then, why do we need to expose `./bacardi lint` command to user?

If `./bacardi format` comand doesn't cover all lint errors, it should be exposed. But now I do not think there is a good reason to expose it. IMHO, it's better to expose a small number of things to users if possible.

Reply:
@nadongguri I left some comments in the ISSUE.